### ISSUE TITLE
wrf-hydro compilation error

### ISSUE THREAD

**Issue by wuesh** [NCAR/wrf_hydro_nwm_public#763]
Hi everyone, I had an error while installing and compiling wrf-hydro. When I installed all the required dependencies, I compiled wrf-hydro and got an error. The error content is in compile.txt 
[compile.txt](https://github.com/user-attachments/files/16386239/compile.txt)
.In fact, I have successfully installed and piloted WRF-Hydro and WRF on my PC, but when I remotely installed it on the server, it failed, and I do not know why this error occurred.
Any help is greatly appreciated!

**Comment by scrasmussen** [comment]
Hi, I think you may be using the older way to build that we are phasing out. The preferred method to install is [with CMake](https://github.com/NCAR/wrf_hydro_nwm_public/blob/main/docs/BUILD.md). Could you try the following from the top directory of the project?

```
$ mkdir build
$ cd build
$ cmake ..
$ make -j 4
```

**Comment by wuesh** [comment]
> Hi, I think you may be using the older way to build that we are phasing out. The preferred method to install is [with CMake](https://github.com/NCAR/wrf_hydro_nwm_public/blob/main/docs/BUILD.md). Could you try the following from the top directory of the project?
> 
> ```
> $ mkdir build
> $ cd build
> $ cmake ..
> $ make -j 4
> ```

Thank you very much. According to the error message, I searched the Internet for possible causes and finally determined that the problem was the installation of the MPICH dependency package. Through the "mpirun -version" command, I found that the mpich dependency package was compiled by ifort, but strangely, I did not modify the contents of FC and FCFLAGS, and checked the compilation logs of each dependency package. Only the MPICH dependency package had a problem, which may be caused by some Settings on the server. I finally reinstalled the MPICH dependency package and successfully compiled the WRF executable.
Thank you very much for your help, and I also hope that my mistake can help friends who have similar problems.
